lib/Spellbook/Exploit/CVE_2021_41773.pm

```diff
@@ -31,7 +35,7 @@ package Spellbook::Exploit::CVE_2021_41773 {
 
             my $useragent = LWP::UserAgent -> new();
             my $request   = $useragent -> get(
-                $target . $payload, 
+               "https://" . $target . $payload, 
```

Comment:
I need remove this concatenate with Https, because on line 20~23 I made another function to the same objective